### ISSUE TITLE
Error: Vagrant::Util::Subprocess::LaunchError

### ISSUE THREAD

**Issue by jadbaz** [bertramn/ansible-win-wrapper#2]
So I managed to run ansible on Cygwin.
The command `ansible-playbook smoke_test.yml` runs flawlessly.
However, Vagrant integration still isn't working.
I think I followed all instructions to the letter.
One thing I did different is, I didn't use `cygwin.ini` but rather just replaced `c:\cygwin` with `c:\cygwin64` in [ansible-playbook.bat](https://github.com/bertramn/ansible-win-wrapper/blob/master/ansible-playbook.bat#L7)

Environment:

- Windows 10
- CYGWIN_NT-10.0 LLBN01CG72549HP 2.10.0(0.325/5/3) 2018-02-02 15:16 x86_64 Cygwin
- VirtualBox 5.2.14
- Vagrant 2.1.2
- Ansible 2.6.2

Files:
- [ansible-playbook.exe](https://github.com/bertramn/ansible-win-wrapper/files/2273634/ansible-playbook.exe.txt)
- [host.rb](https://github.com/bertramn/ansible-win-wrapper/files/2273655/host.rb.txt)
- [Vagrantfile](https://github.com/bertramn/ansible-win-wrapper/files/2273632/Vagrantfile.txt)
- [ansible.cfg](https://github.com/bertramn/ansible-win-wrapper/files/2273633/ansible.cfg.txt)
- [ansible-inventory.yml](https://github.com/bertramn/ansible-win-wrapper/files/2273656/ansible-inventory.yml.txt)
- [smoke_test.yml](https://github.com/bertramn/ansible-win-wrapper/files/2273654/smoke_test.yml.txt)

Error log: [vagrant_ansible_windows_povision_verbose_error_log.txt](https://github.com/bertramn/ansible-win-wrapper/files/2273631/vagrant_ansible_windows_povision_verbose_error_log.txt)

**Comment by bertramn** [comment]
Yeah don't use the `.bat` file. I tried to pull together all the wizardry I found on the web but doing this with batch files is not working. I then wrote the win wrapper to delegate the ansible-playbook call that vagrant would perform to the cygwin environment. Thats the code in the .go file. Checkout post #1.
